reward/staking_manager.go

```diff
@@ -374,6 +430,15 @@ func getStakingInfoFromCache(blockNum uint64) *StakingInfo {
 		return cachedStakingInfo.(*StakingInfo)
 	}
 
+	if info, ok := stakingManager.preloadedInfo.Get(blockNum); ok {
+		info := info.(*StakingInfo)
+		// Fill in Gini coeff if not set. Modifies the cached object.
+		if err := fillMissingGiniCoefficient(info, blockNum); err != nil {
```

Comment:
How about adding a `Preloaded hit ...` with a consistency similar to the one above?

Reply:
Added in c1e0bcdd.